src/components/main-menu/MainMenu.tsx

```diff
@@ -1,13 +1,13 @@
 import * as React from "react";
 import { useHistory } from "react-router-dom";
-import { i18n } from "../../Locale";
+import { i18n } from "@/Locale";
 import "./MainMenu.scss";
-import MenuEntry, { IMenuEntry } from "./menu-entry/MenuEntry";
+import MenuEntry from "./menu-entry/MenuEntry";
 import RateApp from "../rate-app/RateApp";
 import Atom from "@/components/atom";
 
 function MainMenu() {
-  const menuEntries: IMenuEntry[] = [
+  const menuEntries = [
```

Comment:
this is a change in semantics, previous version was saying that menuEntries needs to be of some type and if any item is not, report an error in the item

now, we are saying that menuEntries is a literal type, an array of some objects... and when it will be assigned somewhere else in the future, TS will check whether the literal type can be assigned to that other type and report an error there

is that desired?

Reply:
My logic was that is is defined and used used within this component, so the error wont have moved far.  

One of the major selling features of TS (IMO) is that type inference and duck typing.  Generally, not defining a strict type for a variable will allow for easier refactoring in the future.